src/websocket/server.cc

```diff
@@ -126,6 +143,28 @@ future<> connection::process() {
 
 static std::string sha1_base64(std::string_view source) {
     unsigned char hash[20];
+
+#ifdef SEASTAR_WITH_TLS_OSSL
+    const auto encode_capacity = [](size_t input_size) {
+        return (((4 * input_size) / 3) + 3) & ~0x3U;
```

Comment:
where does this maths come from?

Reply:
https://github.com/redpanda-data/redpanda/blob/52a68fcb0e3a1004c9a63334067344b1b20a47a7/src/v/utils/base64.cc#L21-L24